docs/enterprise-starter.mdx

```diff
@@ -0,0 +1,67 @@
+# Enterprise Starter
+
+<p className="subtitle">Learn about the Enterprise Starter plan tailored for teams and individuals.</p>
+
+<Callout type="info">Enterprise Starter is available for Sourcegraph versions 6.0+ and supports VS Code, JetBrains, Cody Web.</Callout>
+
+The Enterprise Starter plan offers a multi-tenant Sourcegraph designed for small teams and individuals that require the same core features of a traditional Sourcegraph instance but with a simplified management experience. This plan provides a fully managed version of Sourcegraph (AI + code search + omnibox, with privately indexed code) through a self-serve flow.
+
+## Team seats
+
+The Enterprise Starter plan is priced at **$19 per month per seat**. You can add or remove team members at any time. Existing Cody Pro users can also sign up for the Enterprise Starter by paying $19 per seat. However, their Cody Pro subscription will neither be upgraded nor canceled. Instead, they will have two live subscriptions.
+
+## Enterprise Starter team roles
+
+The Enterprise Starter plan includes the following team-level roles:
+
+- **Admin**: Has full access to the workspace, including the ability to manage users, billing, and settings
```

Comment:
Should we explicitly say admin can manage the repos?

Reply:
Yes definitely. 👍